infrateam_first_responder.md

```diff
@@ -10,11 +10,18 @@ The DLSS Infrastructure team is using a rotating role of "first responder." This
   * [Weekly Dependency Updates](#weekly-dependency-updates)
     * [Merge 'em](#merge-em)
     * [Deploy 'em](#deploy-em)
-      * [Important Exceptions](#important-exceptions)
-      * [Code that isn't a Ruby Application](#code-that-isnt-a-ruby-application)
+      * [1. Create a release tag](#1-create-a-release-tag)
+      * [2. Deploy to stage](#2-deploy-to-stage)
+      * [3. Run integration tests in stage](#3-run-integration-tests-in-stage)
+      * [4. Deploy to QA](#4-deploy-to-qa)
+      * [5. Deploy to prod](#5-deploy-to-prod)
+    * [Run infrastructure-integration-tests](#run-infrastructure-integration-tests)
+    * [Required Additional Deploys](#required-additional-deploys)
+      * [Ruby Projects with non-standard Capistrano targets](#ruby-projects-with-non-standard-capistrano-targets)
```

Comment:
```suggestion
```

i think this entry is no longer needed, since the section it links to was removed?

Reply:
@jmartin-sul Good eye! Committing this.